bindings/python/test/test_mongocrypt.py

```diff
@@ -64,15 +66,46 @@ def test_mongocrypt_binary_out(self):
 
 class TestMongoCryptOptions(unittest.TestCase):
 
-    def test_mongocrypt_options(self):
+    def test_mongocrypt_options_nonlocal_kms(self):
         schema_map = bson_data('schema-map.json')
         valid = [
             ({'aws': {'accessKeyId': '', 'secretAccessKey': ''}}, schema_map),
-            ({'aws': {'accessKeyId': 'foo', 'secretAccessKey': 'foo'}}, None),
+            ({'aws': {'accessKeyId': 'foo', 'secretAccessKey': 'foo'}}, None)]
+        for kms_providers, schema_map in valid:
+            opts = MongoCryptOptions(kms_providers, schema_map)
+            self.assertEqual(opts.kms_providers, kms_providers)
+            self.assertEqual(opts.schema_map, schema_map)
+
+    @unittest.skipIf(PY3, 'python 2 only')
+    def test_mongocrypt_options_local_kms_py2(self):
```

Comment:
Can we also test that `{'key': bson.binary.Binary(b'1'*96)}` works correctly on both Python 2 and 3?

Reply:
Done.